### PR TITLE
Add [Tugwa Victor Chinenye] to Rookie list

### PR DESCRIPTION
**Pull request by Vickysnipe**
# Description

Welcome! We are excited to have you on board. This is your first formal step in making it official that you are a part of the Polkadot Heroes program as either a hero or a rookie. Please fill out your profile details in this pull request. Remember to place your profile in alphabetical order in the list, add your profile image to the `./images` subfolder of either the `Heroes/` or `Rookies/` folder, and fill out all the informational fields in your card. You can copy and paste someone else's card and replace their information with your own.

Make sure to preview your additions in the Heroes/README.md or the Rookies/README.md file to confirm it looks as expected.

# Checklist:

- [x] My name is alphabetically ordered by first name in the README
- [x] My profile picture is added to the `/images` folder of either the `Heroes/` or the `Rookies/` directory
- [x] All the fields are filled out for my profile (see the other profiles for examples)

### CODE REVIEW

**Comment by Vickysnipe** [comment]
Add Victor Chinenye Tugwa to the Rookie list

File: Rookies/README.md [hunk 1]
```diff
@@ -196,6 +196,17 @@ You can read the [Heroes Review Criteria and Scoring Matrix Guidelines](../Heroe
 ### Suraj Vijayan
 <img src="./images/suraj.jpg" width="100"/>
 
+### Tugwa Chinenye Victor
```
Comment (hummusonrails) [inline Rookies/README.md:199]: Hey @Vickysnipe,

Can you please make sure to fix this and ensure that your profile addition is separated out from Suraj's? Right now you have inserted your profile right in the midst of theirs.

It should be in alphabetical order by first name, so yours should come after the last "S" first name profile.

File: Rookies/README.md [hunk 2]
```diff
@@ -207,15 +216,6 @@ You can read the [Heroes Review Criteria and Scoring Matrix Guidelines](../Heroe
   - LinkedIn: [Victor Chinenye Tugwa] (https://www.linkedin.com/in/victor-tugwa-a26a68105/)
```
[the remark this answers is not in the document]
Comment (hummusonrails) [reply Rookies/README.md:216]: In Markdown, you need to make sure there is not a space between the `[text]` and the `(link)` parts of the text to turn it into a link. Can you please remove the space in each of your profile links Markdown?

**Comment by hummusonrails** [comment]
Welcome, @Vickysnipe! 🎉

**Pull request merged**